---
title: Best practices for single-cell analysis across modalities
authors:
- Lukas Heumos
- Anna C. Schaar
- Christopher Lance
- Anastasia Litinetskaya
- Felix Drost
- Luke Zappia
- Malte D. LÃ¼cken
- Daniel C. Strobl
- Juan Henao
- Fabiola Curion
- Herbert B. Schiller
- Fabian J. Theis
date: '2023-08-01'
publishDate: '2024-11-15T17:08:50.395678Z'
publication_types:
- article-journal
publication: '*Nature Reviews Genetics*'
doi: 10.1038/s41576-023-00586-w
abstract: Recent advances in single-cell technologies have enabled high-throughput
  molecular profiling of cells across modalities and locations. Single-cell transcriptomics
  data can now be complemented by chromatin accessibility, surface protein expression,
  adaptive immune receptor repertoire profiling and spatial information. The increasing
  availability of single-cell data across modalities has motivated the development
  of novel computational methods to help analysts derive biological insights. As the
  field grows, it becomes increasingly difficult to navigate the vast landscape of
  tools and analysis steps. Here, we summarize independent benchmarking studies of
  unimodal and multimodal single-cell analysis across modalities to suggest comprehensive
  best-practice workflows for the most common analysis steps. Where independent benchmarks
  are not available, we review and contrast popular methods. Our article serves as
  an entry point for novices in the field of single-cell (multi-)omic analysis and
  guides advanced users to the most recent best practices.
tags:
- Functional genomics
- Machine learning
- RNA sequencing
- Software
links:
- name: URL
  url: https://www.nature.com/articles/s41576-023-00586-w
---
